packages/components/typography/src/Heading.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useContext } from 'react';
+import tokens from '@contentful/f36-tokens';
+import {
+  Box,
+  CommonProps,
+  MarginProps,
+  PolymorphicComponent,
+  PolymorphicComponentProps,
+  PolymorphicComponentWithRef,
+} from '@contentful/f36-core';
+import { TypographyContext } from './Typography';
+
+export type HeadingElement = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'p';
```

Comment:
What about `span` and `div`? I don't think I see in issue in opening up for all elements here.

Reply:
Yeah, I guess we can do that. The current type is just a copy of the previous implementation.

Reply:
updated